File: api/Medico.Api/DB/Models/SignatureInfo.cs
using System;

namespace Medico.Api.DB.Models
{
    public class SignatureInfo
    {
        public Guid Id { get; set; }

        public Guid EmployeeId { get; set; }

        public Employee Employee { get; set; }

        public Guid AdmissionId { get; set; }

        public Admission Admission { get; set; }

        public DateTime SignDate { get; set; }

        public bool IsUnsigned { get; set; }
    }
}

File: client/src/dataModels/signatureInfo.ts
import { ConvertibleFromEntityModel } from "./baseEntityModel";
import { DateConverter } from "../helpers/dateConverter";
import { GuidHelper } from "../helpers/guidHelper";

export class SignatureInfo extends ConvertibleFromEntityModel {
    Id: string;
    EmployeeId: string;
    AdmissionId: string;
    SignDate: any;
    IsUnsigned: boolean;

    constructor() {
        super();
        this.Id = GuidHelper.generateNewGuid();
        this.EmployeeId = "";
        this.AdmissionId = "";
        this.SignDate = new Date();
        this.IsUnsigned = false;
    }

    convertToEntityModel() {
        this.SignDate = DateConverter
            .jsLocalDateToSqlServerUtc(this.SignDate);
    }

    createFromEntityModel(entity: any) {
        const signatureInfo = new SignatureInfo();

        signatureInfo.Id = entity.Id;
        signatureInfo.EmployeeId = entity.EmployeeId;
        signatureInfo.AdmissionId = entity.AdmissionId;
        signatureInfo.IsUnsigned = !!entity.IsUnsigned;

        signatureInfo.SignDate = DateConverter
            .sqlServerUtcDateToLocalJsDate(entity.SignDate);

        return signatureInfo;
    }
}